service-onboarding-build-pack/Jenkinsfile

```diff
@@ -60,7 +60,7 @@ node  {
 	def var_bitbucket_branch_perm	= repo_protocol + repo_base + "/rest/branch-permissions/2.0/projects/" + env.REPO_LOC + "/repos/"
 	def var_bitbucket_webhook_api	= repo_protocol + repo_base + "/rest/webhook/1.0/projects/" + env.REPO_LOC + "/repos/"
 	def jenkins_url                 = JenkinsLocationConfiguration.get().getUrl()
-	def bitbucket-scmscource-hook  	= jenkins_url + "/bitbucket-scmsource-hook/notify"
+	def bitbucket_scmscource_hook  	= jenkins_url + "/bitbucket-scmsource-hook/notify"
```

Comment:
rename to scmsource_hook (we are no longer bitbucket specific)

Reply:
Changed